Doc/library/ssl.rst

```diff
@@ -2032,6 +2032,15 @@ to speed up repeated connections from the same clients.
 
    .. versionadded:: 3.7
 
+.. attribute:: SSLContext.security_level
+
+   The current security level for the context. This attribute is read-only.
```

Comment:
Please mention that the security level is an integer and link to https://www.openssl.org/docs/manmaster/man3/SSL_CTX_get_security_level.html